05.ls/lib/core.rb

```diff
@@ -13,6 +13,8 @@ def entries
         [File.basename(path)]
       else
         entries = Dir.entries(path).sort
+        entries = entries.reverse if @option.sort_reverse?
+
```

Comment:
次のコードだと親ディレクトリ（ `..` ）は表示されない🤔
See： https://github.com/maeda-m/ruby-practices/pull/6/files#r847749369

`Dir.pwd/*` となっているので、親ディレクトリが含まれていないのは自然かもしれない気がしてきた🙄

```suggestion
        fnm_flag = @option.ignore_minimal? ? 0 : File::FNM_DOTMATCH
        entries = Dir.glob(File.join(path, '*'), fnm_flag).map { |file_path| File.basename(file_path) }
        entries = entries.reverse if @option.sort_reverse?

        entries
```

Reply:
おや、そうなんですね。私の環境だと、irbで以下のように表示されるのですが...同じ状況でしょうか？ 🤔 

```ruby
irb(main):001:0> Dir.glob("#{Dir.pwd}/*", File::FNM_DOTMATCH)
=> ["/Users/yuhei/ghq/.", "/Users/yuhei/ghq/..", "/Users/yuhei/ghq/github.com"]
```

Reply:
確認いただきありがとうございます🙏

なるほどー。調べたところ挙動が変わったようですね💡

https://bugs.ruby-lang.org/issues/18436
https://github.com/ruby/ruby/commit/5917f49ad03cd3b2c13d3eea85c969a113b83bf8